crates/libtiny_tui/src/tests/config.rs

```diff
@@ -0,0 +1,101 @@
+use libtiny_common::{ChanName, ChanNameRef};
+
+use crate::config::*;
+use crate::notifier::Notifier;
+
+#[test]
+fn parsing_tab_configs() {
+    let config_str = r##"
+        servers:
+          - addr: "server"
+            join: 
+              - "#tiny -ignore -notify messages"
```

Comment:
I don't like this syntax in config, it's inconsistent with how we set the same things for servers and globally.

We should use the same `notifier` and `ignore` fields for all.

Maybe something like:

```
join:
  - name: "#tiny"
    ignore: false
    notifier: off
```

The current syntax should also be supported.

We should also use same naming in the config files and commands. So "notify" instead of "notifier".

Reply:
Yes, fair enough. I will do this.

Reply:
It looks like this was done. Resolving.